### PR TITLE
Remove accidental precommit exclusions

### PR DESCRIPTION
**Pull request by exhaustin**
## Motivation and Context

We accidentally excluded all files from black and isort by removing excluded directories in #63
This PR reverts the change.

## Changelog

- Fix pre_commit_config.yaml to not exclude everything
- TODO: Run pre-commit and clean up files once outstanding branches are merged.

## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Comment by cpaxton** [comment]
Will this break a lot of our existing CI?

**Comment by exhaustin** [comment]
> Will this break a lot of our existing CI?

More like, this will re-enable part of CI against a lot of our code, causing it to fail.
However only formatting is affected, so we can fix the entire codebase with one command.
However(2) that will cause merge conflicts everywhere, so I'd like to wait until the current PRs are merged to make the fix.

**Review by cpaxton — APPROVED** [review]
lgtm, thanks for doing this!

**Pull request merged**
